helm/aladdin-demo/templates/aladdin-demo.configMap.yaml

```diff
@@ -0,0 +1,31 @@
+apiVerson: v1
+kind: ConfigMap
+metadata: 
+  name: {{ .Chart.Name }}-config
+  labels:
+    project: {{ .Chart.Name }}
+    name: {{ .Chart.Name }}-config
+    app: {{ .Chart.Name }}
+# Create key - value entries in the data map that other files can look up
+data:
+  REDIS_HOST: {{ .Chart.Name }}-redis
+  REDIS_PORT: {{ .Values.redis.port | quote }}
+
+---
+# ConfigMap for nginx
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-nginx-config
+data:
+  # desired name for the file
+  nginx.conf: {{ include "nginx-config" . | quote }}
+
+---
+# ConfigMap for uwsgi
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-uwsgi-config
+data:
+  uwsgi.yaml: {{ include "uwsgi-config" . | quote }}
```

Comment:
Not sure if we want to make the standard separating each of these into separate files or putting them all in one